### ISSUE TITLE
Various Peloton HA Integration Errors in Log

### ISSUE THREAD

**Issue by markus99** [edwork/homeassistant-peloton-sensor#75]
Just upgraded to 2023.2.4 and seeing these pop up in the logs:

```
Detected integration that called async_setup_platforms instead of awaiting async_forward_entry_setups; this will fail in version 2023.3. Please report issue to the custom integration author for peloton using this method at custom_components/peloton/__init__.py, line 98: hass.config_entries.async_setup_platforms(entry, PLATFORMS)
```

```
Logger: homeassistant.components.sensor
Source: components/sensor/__init__.py:503
Integration: Sensor (documentation, issues)
First occurred: 9:00:02 AM (3 occurrences)
Last logged: 9:00:02 AM

Entity sensor.mark_on_peloton_start_time (<class 'custom_components.peloton.sensor.PelotonStatSensor'>) is using state class 'measurement' which is impossible considering device class ('timestamp') it is using; expected None; Please update your configuration if your entity is manually configured, otherwise report it to the custom integration author.
Entity sensor.mark_on_peloton_end_time (<class 'custom_components.peloton.sensor.PelotonStatSensor'>) is using state class 'measurement' which is impossible considering device class ('timestamp') it is using; expected None; Please update your configuration if your entity is manually configured, otherwise report it to the custom integration author.
Entity sensor.mark_on_peloton_power_output (<class 'custom_components.peloton.sensor.PelotonStatSensor'>) is using state class 'measurement' which is impossible considering device class ('energy') it is using; expected None or one of 'total_increasing', 'total'; Please update your configuration if your entity is manually configured, otherwise report it to the custom integration author.
```

**Comment by RobertD502** [comment]
Already fixed in #63 and #66!

**Comment by edwork** [comment]
Hello,

Please pardon me for the delay on a full release - this has officially been published as a release under version 0.10.0 and can be installed via HACS. 

Thanks,
Ed
